### PR TITLE
Update appveyor.yml

### PR DESCRIPTION
**Pull request by aymanhab**
choose BTK as OPENSIM_C3D_PARSER as needed by builds downstream

Fixes issue #<issue_number>

### CHANGELOG.md (choose one)

- no need to update because...
- updated...

The Doxygen for this PR can be viewed at http://myosin.sourceforge.net/?C=N;O=D; click the folder whose name is this PR's number.

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/2739)

### CODE REVIEW

**Comment by aymanhab** [comment]
@chrisdembia Any clues why BTK is not being found now when we set OPENSIM_C3D_PARSER=BTK?

**Comment by chrisdembia** [comment]
We set superbuild to not build BTK by default. You must pass `-DSUPERBUILD_BTK=ON` for the superbuild. 

I would suggest using ezc3d instead, though.

**Comment by aymanhab** [comment]
Thanks @chrisdembia I was trying to reduce the unknowns since we have WITH_BTK all over the place, will get the build to go with BTK first then we can test switching to ezc3d.

**Comment by chrisdembia** [comment]
> I was trying to reduce the unknowns since we have WITH_BTK all over the place, will get the build to go with BTK first then we can test switching to ezc3d.

Okay; my concern is we will delay the switch.

**Comment by aymanhab** [comment]
BTK was actually built and testing C3DFileAdapter passed, so I'd like to merge if possible. 
I also created a branch to switch from BTK to EZC3D that can be PR'd after successful build of master (and GUI). Thanks @chrisdembia

File: appveyor.yml [hunk 1]
```diff
@@ -105,7 +105,7 @@ build_script:
   # Configure. # TODO -DBUILD_SIMM_TRANSLATOR=ON
   # Set the CXXFLAGS environment variable to turn warnings into errors.
   # Skip timing test section included by default
-  - cmake -E env CXXFLAGS="/WX -DSKIP_TIMING" cmake %OPENSIM_SOURCE_DIR% -G"%CMAKE_GENERATOR%" -T"%CMAKE_TOOLSET%" -DSIMBODY_HOME=C:\simbody%NUGET_PACKAGE_ID_SUFFIX% -DOPENSIM_DEPENDENCIES_DIR=%OPENSIM_DEPENDENCIES_INSTALL_DIR% -DCMAKE_INSTALL_PREFIX=%OPENSIM_INSTALL_DIR% -DBUILD_JAVA_WRAPPING=ON -DBUILD_PYTHON_WRAPPING=ON -DWITH_BTK:BOOL=ON
+  - cmake -E env CXXFLAGS="/WX -DSKIP_TIMING" cmake %OPENSIM_SOURCE_DIR% -G"%CMAKE_GENERATOR%" -T"%CMAKE_TOOLSET%" -DSIMBODY_HOME=C:\simbody%NUGET_PACKAGE_ID_SUFFIX% -DOPENSIM_DEPENDENCIES_DIR=%OPENSIM_DEPENDENCIES_INSTALL_DIR% -DCMAKE_INSTALL_PREFIX=%OPENSIM_INSTALL_DIR% -DBUILD_JAVA_WRAPPING=ON -DBUILD_PYTHON_WRAPPING=ON -DOPENSIM_C3D_PARSER=BTK -DWITH_BTK:BOOL=ON
```
Comment (chrisdembia) [inline appveyor.yml:108]: WITH_BTK should be removed.

Comment (aymanhab) [reply appveyor.yml]: Done, if the build succeeds will merge. Thanks @chrisdembia

**Comment by chrisdembia** [comment]
Thanks @aymanhab

**Pull request merged**